pkgs/development/libraries/gle/default.nix

```diff
@@ -27,12 +28,15 @@ stdenv.mkDerivation rec {
     libXext
   ];
 
-  src = fetchurl {
-    urls = [
-      "mirror://sourceforge/project/gle/gle/gle-${version}/gle-${version}.tar.gz"
-      "https://www.linas.org/gle/pub/gle-${version}.tar.gz"
-      ];
-    sha256 = "09zs1di4dsssl9k322nzildvf41jwipbzhik9p43yb1bcfsp92nw";
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  src = fetchFromGitHub {
+    owner = "linas";
+    repo = "glextrusion";
+    rev = "refs/tags/${pname}-${version}";
+    sha256 = "sha256-yvCu0EOwxOMN6upeHX+C2sIz1YVjjB/320g+Mf24S6g=";
   };
```

Comment:
Why move src down? Usually it goes before inputs.

Reply:
Remained in place relative to inputs, wasn't moved

Reply:
Oh my mistake — you're right!